content/docs/traffic-management/weighted-routes.md

```diff
@@ -0,0 +1,414 @@
+---
+title: Route weighting
+weight: 20
+---
+
+By default, the Kubernetes Gateway API sorts HTTPRoute rules based on their order and specificity, as defined in the [Gateway API docs](https://gateway-api.sigs.k8s.io/reference/spec/#httprouterule). When a route matches a request, no further routes are evaluated for matching, which might affect the policies and final destination of the traffic. With {{< reuse "/docs/snippets/kgateway.md" >}}, you can configure weights for more fine-grained control over your routing rules.
+
+## Before you begin
+
+{{< reuse "docs/snippets/prereq.md" >}}
+
+## Step 1: Review default routing behavior {#default-routing}
+
+By default, the Kubernetes Gateway API sorts HTTPRoute rules as defined in the [Gateway API docs](https://gateway-api.sigs.k8s.io/reference/spec/#httprouterule).
+
+1. Specificity of the route matching rule, such as a header match with largest number of matching headers.
+2. Oldest route based on creation timestamp.
+3. Alphabetical order of the route based on the `namespace/name` of the route.
+4. Order of the route in the list of the HTTPRoute resource.
+
+Review route precedence in action:
+
+1. Deploy a hello-world app to demonstrate different backend responses.
```

Comment:
not sure if there is a better app to use, basically i wanted two different app responses for the two different paths so that later, you could see it going to the higher-weighted path's backing app

Reply:
yeah for such cases, I am using the hello world app too